src/onTTFB.ts

```diff
@@ -82,12 +82,16 @@ export const onTTFB = (onReport: ReportCallback, opts?: ReportOpts) => {
       metric.entries = [navEntry];
 
       report(true);
-    }
-  });
 
-  onBFCacheRestore(() => {
-    metric = initMetric('TTFB', 0);
-    report = bindReporter(onReport, metric, thresholds, opts!.reportAllChanges);
-    report(true);
+      // Only report TTFB after bfcache restores if a `navigation` entry
```

Comment:
@tunetheweb FYI: I moved this `onBFCacheRestore()` call inside the check for the navigation entry to ensure that a TTFB metric wasn't reported after a bfcache restore in cases where the browser doesn't support the Navigation Timing API at all.

This still wouldn't affect cases where `navigationType` was undefined because in bfcache restores the type is set to `back-forward-cache`, but I do think it was a bug since browsers like Opera mini in extreme data saver mode would have run into this.

I checked all the other functions to, but they were all fine since they're gated on support for `PerformanceObserver`.